pkg/resolvconf/resolvconf_test.go

```diff
@@ -0,0 +1,201 @@
+package resolvconf
```

Comment:
This is its own package because it's very decoupled from runtime needs.
I couldn't find a more sensible home for it.

Reply:
Yes. I have no problem with this in a separate package :-)